EVGEN/AliGenPerformance.cxx

```diff
@@ -88,35 +90,35 @@ AliGenPerformance::AliGenPerformance():
 }
 //-----------------------------------------------------------------------------
 AliGenPerformance::AliGenPerformance(const char* generName, Int_t verboseLevel):
-  AliGenerator(),
-  fNJets(1),                // mean number of jets per event
-  fF1Momentum(0),           // momentum distribution function 
-  fFPhi(0),                 // phi distribution function
-  fFTheta(0),               // theta distribution function
-  fFPosition(0),            // position distribution function 
-  fFPdg(0),                 // pdg distribution function  
-  fStreamer(0),           // test stream - used for tuning of parameters of generator
-  fVerboseLevel(0)            // verbose level
+        AliGenerator(),
+        fNJets(1),                // mean number of jets per event
+        fF1Momentum(0),           // momentum distribution function
+        fFPhi(0),                 // phi distribution function
+        fFTheta(0),               // theta distribution function
+        fFPosition(0),            // position distribution function
+        fFPdg(0),                 // pdg distribution function
+        fStreamer(0),           // test stream - used for tuning of parameters of generator
+        fVerboseLevel(0)            // verbose level
```

Comment:
This is just indentation according to Clion. The real changes are very few. You can see them if you add ?w=1 at the end of the URL